CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
 this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Added `ranked_outlier_stream` Graphql API to fetch `ranked outlier` periodically.
+  - Gets the `model id` of the currently stored `model`.
```

Comment:
Is `model id` a proper code snippet?

Reply:
Modified the CHANGELOG documentation to use appropriate code snippets.